src/python/grpcio_tests/tests_py3_only/interop/xds_interop_client.py

```diff
@@ -150,8 +150,8 @@ def GetClientStats(
         return response
 
     def GetClientAccumulatedStats(
-            self, request: messages_pb2.LoadBalancerAccumulatedStatsRequest,
-            context: grpc.ServicerContext
+        self, request: messages_pb2.LoadBalancerAccumulatedStatsRequest,
```

Comment:
The changes in this file don't seem to be related to the rest of the PR.  If this is something that's broken in master, let's fix that in a separate PR before merging this one.

Reply:
Good idea, I've removed it from this PR. From the yapf upgrade yesterday, I'm guessing we may expect more formatting changes as we go along.

Reply:
There shouldn't be more changes needed.  This was probably caused by two PRs being in flight at the same time -- the yapf change and another one that added a new file.  It shouldn't happen on an ongoing basis.

Reply:
Jan fixed the formatting issue in e2b17027